### ISSUE TITLE
Need a way to clear combox input from another component

### ISSUE THREAD

**Issue by eddielca** [IBM/carbon-components-angular#1028]
## Detailed description
we would like to clear input by clicking on `clear search` button, however, since this function is not callable either by using template or event emitter, we can't clear it. is it possible to expose the function to at least let us call using viewchild?
![Screen Shot 2020-01-31 at 11 41 32 AM](https://user-images.githubusercontent.com/47797763/73557222-d4e97a00-441e-11ea-85c0-92ba69416f3b.png)

> Describe in detail the issue you're having.

> Is this a feature request (new component, new icon), a bug, or a general issue?

> Is this issue related to a specific component?

> What did you expect to happen? What happened instead? What would you like to see changed?

> What browser are you working in?

> What version of the Carbon Design System are you using?

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

* Screenshots or code
* Notes

**Comment by cal-smith** [comment]
Agreed, we should definitely expose a `value`/`valueChange` input/output ... in the interim try `clearInput()` with an empty event (`clearInput(new Event())` or something similar should work)
